jsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "compilerOptions": {
+    "baseUrl": "src"
+  }
+}
```

Comment:
TIL https://code.visualstudio.com/docs/languages/jsconfig

Reply:
@mattbrictson  -- that solution was brilliant! #TIL